### ISSUE TITLE
multiple clipboard formats

### ISSUE THREAD

**Issue by mfulton26** [glawrence/titles-extension#18]
I would like to be able to copy both rich text and plain text formats at once

that way if I paste it into Word, Slack, etc. I will get a formatted link with the title of the page as the display text and it linked to the page while if I paste into Markdown or a plain text editor I want to get whatever plain text format I specify (in my case I'll want Markdown)

this way I can quickly copy and paste it in one or multiple places and depending on what the paste target supports I'll get the "best" version pasted

**Comment by glawrence** [comment]
Good idea, I think Windows supports this, but I am not sure the browser APIs do, I will need to  investigate
